components/advocates/MeetTheAdvocates.vue

```diff
@@ -62,3 +62,12 @@ export default class extends Vue {
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import '~/assets/scss/blocks/copy.scss';
```

Comment:
I don't think this is being used

Reply:
I think is needed for the `copy_title` and `copy_paragraph`